app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -126,6 +124,16 @@ message Auth {
     string client_secret = 3;
     string redirect_url_scheme = 4;
   }
+
+  message AllowList {
+    // allow_list is a list of allowed email addresses or domains
+    // for example ["@chainloop.dev", "foo@mycompany.org"]
+    repeated string rules = 1 [(buf.validate.field).string.min_len = 1];
```

Comment:
@javirln did you run this locally? I think its failing since this is a list not a string?